runtime/centrifuge/src/lib.rs

```diff
@@ -1845,6 +1969,7 @@ construct_runtime!(
 		EVMChainId: pallet_evm_chain_id::{Pallet, Config, Storage} = 161,
 		BaseFee: pallet_base_fee::{Pallet, Call, Config<T>, Storage, Event} = 162,
 		Ethereum: pallet_ethereum::{Pallet, Config, Call, Storage, Event, Origin} = 163,
+		EthereumTransaction: pallet_ethereum_transaction::{Pallet, Storage, Event<T>} = 164,
```

Comment:
This si not 3rd party but ours right?

Reply:
Yes, this is ours. It's what we added when adding the gateway.

Reply:
Just as this is o=under 3rd party pallets. But no blocker